packages/plugin-conversation/src/interceptor.js

```diff
@@ -60,9 +60,8 @@ export default class ConversationInterceptor extends Interceptor {
         }
 
         const hasItems = Boolean(response.body.items);
-
-        return this.spark.conversation.decrypter.decryptObject(null, response.body)
-          .then((body) => this.spark.conversation.normalizer.normalize(hasItems ? body.items : body))
+        return this.spark.conversation.decrypter.decryptObject(null, hasItems ? response.body.items : response.body)
```

Comment:
If the response has an items attribute, Decrypter needs the items and not `{items: [...]}`